dbms/src/Server/Server.cpp

```diff
@@ -1174,6 +1174,11 @@ int Server::main(const std::vector<std::string> & /*args*/)
     size_t delta_index_cache_size = config().getUInt64("delta_index_cache_size", 0);
     global_context->setDeltaIndexManager(delta_index_cache_size);
 
+    if (global_context->getSettings().local_tunnel_version == 1)
```

Comment:
This seems meaningless to log local tunnel version here since the setting can be changed without restart TiFlash?

Reply:
> This seems meaningless to log local tunnel version here since the setting can be changed without restart TiFlash?

I have deleted it.